mariadb_skysql/metadata.csv

```diff
@@ -0,0 +1,187 @@
+skysql.maxscale_modules,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_active_operations,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_adaptive_avg_select_time,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_connection_pool_empty,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_connections,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_max_connections,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_max_pool_size,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_persistent_connections,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_reused_connections,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_routed_packets,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_server_total_connections,count,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_service_connections,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_count,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_current_descriptors,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_errors,count,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_event_queue_length,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_hangups,count,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_max_queue_time,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_reads,count,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_stack_size,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_total_descriptors,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_threads_writes,count,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_up,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.maxscale_uptime_seconds,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.process_resident_memory_bytes,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.process_virtual_memory_bytes,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
+skysql.service_node_cpu,gauge,,,,See MariaDB doc at https://mariadb.com/kb/en/,,mariadb_skysql,,,
```

Comment:
I'm unsure of how this will appear in-app and on the Docs Site. The metrics table will not generate a link for these descriptions + the link will be cut off when a user looks at Metrics tab on the integration tile.